flor/journal/file.py

```diff
@@ -1,13 +1,21 @@
 from .entry import DataVal, DataRef, Bracket, EOF, make_entry
 
 from flor import shelf
+from ..logger import Logger
 
 import json
 import pathlib
 from typing import Union, List
 
 entries: List[Union[DataRef, DataVal, Bracket, EOF]] = []
 
+def _proc_log_record(log_record):
```

Comment:
Isn't this function already defined elsewhere in the code? Can we avoid code duplication here? Have the code block be defined in a single spot and used in multiple spots.

Reply:
I couldn't find where it's duplicated. Mind pointing me to it?